core/templates/components/summary-tile/story-summary-tile.component.ts

```diff
@@ -237,6 +260,14 @@ export class StorySummaryTileComponent implements OnInit {
     this.getStoryStatus();
   }
 
+  getChapterTextBasedOnCount(chapterCount: number): string {
+    let chapterText = 'Chapters';
+    if (chapterCount === 1) {
+      chapterText = 'Chapter';
+    }
+    return chapterText;
+  }
```

Comment:
Is it possible to use this for pluralization?

https://github.com/oppia/oppia/blob/5806a72fb29101bd81995258dfbe2e00117e8e85/assets/i18n/en.json#L1402

Reply:
@kevintab95 Done. No changes in the UI.